src/main/scala/io/citrine/lolo/bags/Bagger.scala

```diff
@@ -324,15 +333,21 @@ class BaggedResult(
 
     /* These operations are pulled out of the loop and extra-verbose for performance */
     val JMat = NibJ.t * predMat
+    Async.canStop()
     val JMat2 = JMat :* JMat * ((Nib.size - 1.0) / Nib.size)
+    Async.canStop()
     val IJMat = NibIJ.t * predMat
+    Async.canStop()
```

Comment:
No reason not to sprinkle these around liberally, but as long as we can bound time to interrupt below ~1-2secs, it's fine.